README.md

```diff
@@ -21,27 +21,27 @@ Consider this file name:
 What is the purpose of this file, and what is the purpose of the numbers at the beginning of its name?
 
 ```text
-Your answer...
+its a migration file, the numbers at the beginning are a time stamp / method of version control.
 ```
 
 ### Question 3
 
 In a Rails application, how is the router related to controller actions?  
 
 ```text
-Your answer...
+The router takes a query from the browser and directs it to its corresponding controller.
 ```
 
 ### Question 4
 
 Assuming our Tunr Rails app (1) has a Song model that belongs to an Artist model and (2) uses nested resources, which of the following helpers would create a URL that routes to `songs#new`? (Select one answer)  
 
 ```
-[] artist_song_path( @artist, @song ) 
+[x] artist_song_path( @artist, @song )
 [] new_artist_song_path( @artist )
 [] create_artist_song_path( @artist )
 [] new_artist_song_path( @artist, Song.all )
-[] new_song_path( @song ) 
+[] new_song_path( @song )
 ```
```

Comment:
The correct answer is `new_artist_song_path( @artist )`.

`artist_song_path( @artist, @song )` would take you to the show page for a particular song.